src/styles/_button.scss

```diff
@@ -0,0 +1,20 @@
+.button-buy {
+  background: $main-button-color;
+  color: $main-background-color;
+  font-weight: 700;
+  font-size: 14px;
+  text-align: center;
+  line-height: 16px;
+  text-transform: uppercase;
+  width: 166px;
+  height: 40px;
+  margin: 16px 16px 0;
```

Comment:
Add padding for the card and remove the horizontall margin everywhere

Reply:
Do you mean I should remove horizontal margins in all my code or only in the button?

Reply:
fixed